templates/rust-starter/host/src/main.rs

```diff
@@ -12,7 +12,12 @@ fn main() {
     let env = ExecutorEnv::default();
 
     // TODO: add guest input to the executor environment using
-    // ExecutorEnv::add_input()
+    // ExecutorEnvBuilder::add_input().
+    // To access this method, you'll need to use the alternate construction
+    // ExecutorEnv::builder(), which creates an ExecutorEnvBuilder. When you're
+    // done adding input, call ExecutorEnvBuilder::build().
+
+    // For example: let env = ExecutorEnv::builder().add_input(&vec).build();
```

Comment:
Perhaps we should pre-separate this as
```
let env = ExecutorEnv::builder().build();
```
and then we can tell people to add there inputs in the middle.

Reply:
> I can definitely see some of the tradeoffs you were mentioning in getting this tested, but I do think we're all right that the testing is worth those tradeoffs.

Agreed -- other than accepting that a few lines are commented out here and there or have bonus explanatory comments, I feel all right about the final product.

Reply:
> Perhaps we should pre-separate this as
> 
> ```
> let env = ExecutorEnv::builder().build();
> ```
> 
> and then we can tell people to add there inputs in the middle.

Okay great, if there's no reason that `ExecutorEnv::default()` was left in the starter template (and I can't see why there would be, since it's just wrapping `ExecutorEnv::builder().build()`), I'll change both in the README and the starter template.